lib/appium_lib/driver.rb

```diff
@@ -376,6 +378,8 @@ def initialize(opts = {})
       @sauce_username   = nil if !@sauce_username || (@sauce_username.is_a?(String) && @sauce_username.empty?)
       @sauce_access_key = appium_lib_opts.fetch :sauce_access_key, ENV['SAUCE_ACCESS_KEY']
       @sauce_access_key = nil if !@sauce_access_key || (@sauce_access_key.is_a?(String) && @sauce_access_key.empty?)
+      @sauce_endpoint   = appium_lib_opts.fetch :sauce_endpoint, ENV['SAUCE_ENDPOINT']
+      @sauce_endpoint   = "ondemand.saucelabs.com:443/wd/hub" if !@sauce_endpoint || (@sauce_endpoint.is_a?(String) && @sauce_endpoint.empty?)
```

Comment:
> lib/appium_lib/driver.rb:382:27: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
      @sauce_endpoint   = "ondemand.saucelabs.com:443/wd/hub" if !@sauce_endpoint || (@sauce_endpoint.is_a?(String) && @sauce_endpoint.empty?)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/appium_lib/driver.rb:382:126: C: Metrics/LineLength: Line is too long. [142/125]
      @sauce_endpoint   = "ondemand.saucelabs.com:443/wd/hub" if !@sauce_endpoint || (@sauce_endpoint.is_a?(String) && @sauce_endpoint.empty?)
                                                                                                                             ^^^^^^^^^^^^^^^^^

Could you fix rubocop error? @jlipps